app/main/RTMedia.php

```diff
@@ -1746,6 +1746,14 @@ public static function expanded_allowed_tags() {
 			'id'    => array(),
 		);
 
+		// Script, for social sync plugin.
+		$new_allowed['script'] = array(
+			'type'  => true,
```

Comment:
For information: don't we need to pass array()?

Reply:
Used array instead of `true`